instagram/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummyData';
+import PostContainer from './components/PostContainer/PostContainer';
+import CommentSection from './components/CommentSection/CommentSection';
+import SearchBar from './components/SearchBar/SearchBar';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+
+      // brackets?
+      userData: []
+    }
+  }
+
+  componentDidMount() {
+    this.setState({userData: userData})
+  }
+ 
+  render() {
+    return (
+      <div className="App">
+        <SearchBar />
+        {/* {this.state.userData.map(userData => (<PostContainer instagram={this.state.instagram} />  */}
+          {this.state.userData.map(userData => (<PostContainer key={instagram.id} /> 
```

Comment:
Great work mapping over the data! A few things to note here - first, is instagram defined? Instead of looking at the id on instagram, you should be looking at the id of the object that is being passed in through the map function. You are also missing a few parentheses and curly braces on this line